packages/babel-plugin-transform-remove-console/src/index.js

```diff
@@ -3,6 +3,18 @@ export default function () {
     visitor: {
       CallExpression(path) {
         if (path.get("callee").matchesPattern("console", true)) {
+          const statementPath = path.parentPath.parentPath;
```

Comment:
@danez It looks `path.parentPath.parentPath` gets problem in top-level case for remove-console, and I just used `path.parentPath` for remove-debugger.

Anyway, @loganfsmyth's [recommend](https://github.com/babel/babel/pull/3453#issuecomment-206641414) will be better, I'll change it. :)